practice-three/src/js/script.js

```diff
@@ -208,10 +174,41 @@ const handleCreateForm = () => {
   createUser(formData); 
 }
 
+/**
+ * The function takes data from the API and displays it on a table in HTML
+ */
+ const renderUserTable = () => {
+  fetch(userApi)
+    // Parses JSON response into native JavaScript objects 
+    .then((response) => response.json())
+    .then((users) => {
+      let tableTemplate = '';
+         
+      users.forEach(user => {
+        tableTemplate += ` 
+          <tr class="content-row">
+            <td>${user.name}</td>
+            <td>${user.email}</td>
+            <td>${user.salary}</td>
+            <td>${user.city}</td>
+            <td class="td-btn">
+              <button type="button" class="delete-button">Delete</button>
+              <button type="button" class="update-button">Update</button>
```

Comment:
Remove this line, this function haven't implemented yet. So it can not do anything if user click on this button